### PR TITLE
Fix empty a tags for RGAAv3 checkers

### LINKED ISSUE #475: Supprimer les <a> vides
Certaines pages peuvent apparemment avoir des `<a>` vides.

Cela n'a aucun impact réel d'accessibilité mais diminue le score sur les outils automatisés utilisés par certains benchmarks.

### PR DESCRIPTION
**Pull request by Morendil**
Fixes #475 (startup cards)

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -94,3 +94,14 @@ body {
 		line-height: 1.5;
 	}
 }
+
+.sr-only {
+  position: absolute;
```
Comment (MattiSG) [inline css/homepage.css]: Inconsistent whitespace: file uses tabs.

File: css/homepage.css [hunk 2]
```diff
@@ -94,3 +94,14 @@ body {
 		line-height: 1.5;
 	}
 }
+
+.sr-only {
```
Comment (MattiSG) [inline css/homepage.css]: If not using a library, prefer an explicit classname over a bootstrap convention: `screenreader-only`.

File: css/homepage.css [hunk 3]
```diff
@@ -94,3 +94,14 @@ body {
...
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0,0,0,0);
+  border: 0;
+}
```
Comment (MattiSG) [inline css/homepage.css]: Wrong file: this is not specific to the homepage.

File: _includes/startup-presentation.html [hunk 4]
```diff
@@ -11,7 +11,10 @@
 
 <div class="startup-presentation">
 	<div class="image">
-		<a href="{{ description.url }}" title="{{ description.title }}">{% include screenshot.html startup=description %}</a>
+		<a href="{{ description.url }}" title="{{ description.title }}">
+		{% include screenshot.html startup=description %}
+		<span class="sr-only">Page de détail de {{ description.title }}</span>
```
Comment (MattiSG) [inline _includes/startup-presentation.html]: Why “Page de détail de”? Prefer short links, especially when targeting vocalisers: “Détails sur” or “Présentation de” or “Fiche produit de” or “Découvrir”.

File: css/homepage.css (hunk 3 above)
Comment (Morendil) [inline css/homepage.css]: At present it seems to be... but moved to `main.css`.

**Pull request merged**